src/main/java/seedu/address/model/module/Code.java

```diff
@@ -0,0 +1,66 @@
+package seedu.address.model.module;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a Module's code in the transcript.
+ * <p>
+ * Guarantees: immutable; is valid as declared in {@link #isValidCode(String)}
+ */
+public class Code {
```

Comment:
Code class was not mentioned in the Class Diagram, mind updating the class diagram to show the purpose of this class?

Reply:
Yep, going to update documentations for both lucidchart and the developers guide once i'm done adding test.